client/packages/common/src/ui/components/navigation/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -41,6 +41,11 @@ export const Breadcrumbs = ({
       ? t('breadcrumb.item', { id: part.value })
       : t(part.key);
 
+  // Removes the first element if it is a top level path to get correct breadcrumbs
```

Comment:
I rather like having `Reports / ` in the breadcrumb.
If you remove this change and instead do this in the `reports/DetailView/DetailView.tsx`
```
    setCustomBreadcrumbs({ 1: report.name ?? '' });
```

i.e. change the `0` to a `1` then that give this result:

![image](https://github.com/user-attachments/assets/3d0a5b22-c71e-442d-a09a-a926af7d10fc)

Reply:
though sometimes the `useEffect` timing is whack and you get 

![image](https://github.com/user-attachments/assets/0afe9d15-dd48-4fe4-83b7-e2f2927678b7)

so I'd change to 

```
  const { setCustomBreadcrumbs } = useBreadcrumbs(['reports']);
```

in the same file